### PR TITLE
Fix regex path windows

### PR DESCRIPTION
**Pull request by lievendoclo**

### CODE REVIEW

File: src/sdm/binding/project/fileSystemProjectPersister.ts [hunk 1]
```diff
@@ -40,13 +41,13 @@ import { FileSystemRemoteRepoRef } from "./FileSystemRemoteRepoRef";
  * conventions. Perform a git init and other after actions, such as installing
  * our git hooks. Fires off the necessary new repo events using HTTP to
  * all local automation clients.
- * @return {ProjectPersister}
+ * @return {ProjectPersister
```
Comment (johnsonr) [inline src/sdm/binding/project/fileSystemProjectPersister.ts]: missing }

File: test/binding/expandedTreeUtilsTest.ts [hunk 2]
```diff
@@ -14,49 +14,52 @@
...
 describe("expandedTreeUtils", () => {
 
+    const sep = path.sep;
+
     describe("parseOwnerAndRepo", () => {
 
         it("works not within directory tree", () => {
-            const base = "/usr/foo";
-            const dir = base + "/c/d/e";
+            const base = sep + "usr" + sep + "foo";
```
Comment (johnsonr) [inline test/binding/expandedTreeUtilsTest.ts]: Using`path.join` would be prettier

**Comment by lievendoclo** [comment]
Fixes applied

File: test/binding/expandedTreeUtilsTest.ts (hunk 2 above)
Comment (cdupuis) [reply test/binding/expandedTreeUtilsTest.ts]: This should be `path.join(usr, foo)`

**Pull request merged**